testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/Constants.java

```diff
@@ -39,7 +39,8 @@
         INSTANCE_AWARE_COMMON,
         INSTANCE_AWARE_ORGANIZATION,
         B2C,
-        MSA
+        MSA,
```

Comment:
We should consider this making more dynamic through selection in the app and then generating the json on the fly. That would allow us to generate new combinations on need basis rather than having to checkin a file for a specific case.

Reply:
I'll investigate later what are the options for this case. Going to merge this for now.